internal/pkg/warewulfd/daemon.go

```diff
@@ -48,16 +49,14 @@ func DaemonStart() error {
 
 		fmt.Fprintf(p, "%d", pid)
 
-		p.Close()
-
 	}
 
 	return nil
 }
 
 func DaemonStatus() error {
 	if !util.IsFile(WAREWULFD_PIDFILE) {
-		wwlog.Printf(wwlog.INFO, "Warewulf daemon process not running (%s)\n", WAREWULFD_PIDFILE)
+		fmt.Printf("Warewulf daemon process not running (%s)\n", WAREWULFD_PIDFILE)
```

Comment:
This should be returning the error. I'v actually already fixed this in #125.

Reply:
It's not an error though, just a status message.

Thoughts on how to best handle this?

Reply:
Fixed in commit update